pkgs/os-specific/linux/kernel/linux-bcachefs-bio.c-fix.patch

```diff
@@ -0,0 +1,75 @@
+diff --git a/block/bio.c b/block/bio.c
```

Comment:
Can we fetch this from upstream or upstream this?

Reply:
Technically yes, but we get the same error we do if we do it without the patch.
This PR's patch on the left, fetched patch on the right with ``filterdiff -i``.

![Screenshot from 2022-08-02 09-23-50](https://user-images.githubusercontent.com/93990818/182398371-f1f99533-0d1e-4bf1-8fd7-94dddfe60645.png)

Reply:
I don't know why we can't just patch normally either, that's the thing.
It just doesn't work.